### PR TITLE
Add org.kde.haruna

### PR DESCRIPTION
**Pull request by g-fb**
[com.georgefb.haruna](https://flathub.org/apps/details/com.georgefb.haruna) is now part of KDE, therefore it should be renamed.

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions.
  - When a file is opened it adds files from the same folder to the playlist.
  - Users can define folders where for search for subtitles
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project.
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. 

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by g-fb** [comment]
bot, build org.kde.haruna

File: org.kde.haruna.yml [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+sdk: org.kde.Sdk
+command: haruna
+finish-args:
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--socket=pulseaudio'
+  - '--device=dri'
+  - '--filesystem=host'
```
Comment (hfiguiere) [inline org.kde.haruna.yml:14]: do you really host access?

File: org.kde.haruna.yml [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+  - '--talk-name=org.freedesktop.ScreenSaver'
+  - '--own-name=org.mpris.MediaPlayer2.haruna'
+  - '--env=LC_NUMERIC=C'
+modules:
+  - name: haruna
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: 'https://invent.kde.org/multimedia/haruna/-/archive/0.6.3/haruna-0.6.3.tar.gz'
+        sha256: 'c79ec1e351f47faf9a58a6ba7ec3cc05cfdc5423fde0584f2d4081f5058363e3'
```
Comment (hfiguiere) [inline org.kde.haruna.yml:24]: build failure is because it can't find the appstream file.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by g-fb** [comment]
bot, build org.kde.haruna

File: org.kde.haruna.yml (hunk 1 above)
Comment (g-fb) [reply org.kde.haruna.yml:14]: When a file is opened it adds files from the same folder to the playlist.
Users can define folders where for search for subtitles

File: org.kde.haruna.yml (hunk 2 above)
Comment (hfiguiere) [reply org.kde.haruna.yml]: home isn't enough for that? or even xdg-videos?

Comment (g-fb) [reply org.kde.haruna.yml]: No. For example  have an external drive that  gets mounted in `/run/media`.

Comment (g-fb) [reply org.kde.haruna.yml]: @hfiguiere ping

**Comment by barthalion** [comment]
/merge

Please make sure to mark the old ID as EOL: https://github.com/flathub/flathub/wiki/App-Maintenance#end-of-life

**Pull request closed without merging**